bapsflib/_hdf_mappers/map_digitizers/siscrate.py

```diff
@@ -476,7 +476,8 @@ def construct_dataset_name(self, board, channel,
         #   '3302' is not active then the list will only contain '3305'.
         if adc is None:
             adc = self.configs[config_name]['adc'][0]
-            warn_str += '\nNo adc specified, so assuming ' + adc + '.'
+            warn_str += '\n** Warning: no adc specified, so assuming ' \
```

Comment:
remove `'** Warnings:'` from `warn_str`. Since Python's warning system is used, this is addition is redundant.